src/AddressRegistry.Projections.Syndication/LinkedFeedProjectionRunner.cs

```diff
@@ -81,8 +83,16 @@ public class LinkedFeedProjectionRunner<TMessage, TContent, TContext> : ILinkedF
                 position = dbPosition?.Position + 1;
             }
 
-            // Read new events
-            var entries = (await _atomFeedReader.ReadEntriesAsync(FeedUri, position, FeedUserName, FeedPassword, EmbedEvent, EmbedObject)).ToList();
+            var entries = new List<IAtomEntry>();
+            var jitterer = new Random();
+
+            await Polly.Policy
```

Comment:
Not sure if this can be simplified? In this style:

`var something = await policy.ExecuteAsync(async () => await DoSomethingAsync()); // async execute overload`

```
var entries = await Polly.Policy
.Handle<XmlException>()
                .WaitAndRetryAsync(5,
                    retryAttempt => TimeSpan.FromSeconds(Math.Pow(2, retryAttempt)) // exponential back-off: 2, 4, 8 etc
                                    + TimeSpan.FromMilliseconds(jitterer.Next(0, 10000))) // plus some jitter: up to 10 seconds (cause sync request can be heavy)
                .ExecuteAsync(async () => (await _atomFeedReader.ReadEntriesAsync(FeedUri, position, FeedUserName, FeedPassword, EmbedEvent, EmbedObject)).ToList()); // Read new events
```